### ISSUE TITLE
About sticking furniture and some wrong issues

### ISSUE THREAD

**Issue by c469591** [stardew-access/stardew-access#2]
Hello, I will organize the problem,

1. The installation introduction is to register an account from a forum, then download the MOD that sticks furniture, but after I register, I can't download that mod in the forum.
The display has been downloaded on the page, but it does not have any downloads, and I have not allowed me to store the window of the file.
I tried to download the MOD sticking furniture from github, but the downloaded data folder is unable to produce.
Config.json
So I still can't fix my furniture now, I used to take the bed, and finally I can only start a new game.
I hope that the great author can integrate the function of the furniture to the MOD of the game in the future version.
If there is a related option in the game to give the user, it is better, I will thank the author.

2. In addition, I will open the menu in the game, sometimes pressing the ESC will not exit 

3. In the farmland, when I remove the grass or stone, I walked on it, the screen reading soft body won't tell me where this is.
Until I used the hoe to turn over the ground, I will read the soil.
I hope to add some prompts.

4. I switch the game language to Chinese, but I still use English when I read the menu or some projects.
I hope that the great author can provide templates to translate, or provide Chinese language support,
In addition, I use traditional Chinese.

5. Sometimes, from the game main menu, click Load Archive game will not be able to operate,
I can see the date of the game archive, but I can't click on him, and I can't return to the main menu. I can only turn off the game to open.

6. In the game, when I hit the wall, I hope I can use the sound to tell me to hit the wall.
Because walking often holds the keyboard, the text topic of the wall is often interrupted 

7. When you write the name and the name of the farm at the beginning, I often don't know which one I have written. Which one has not written,
When you click OK, I found that there is no reaction, I know that there may be some things have not been written.
So I hope to update better

thanks!

**Comment by shoaib11120** [comment]
Sorry for the late reply. Thanks for the awesome feedback. It will definetly improve the mod. Also maybe join the discord server cause i've been uploading beta versions of the next update which will include more accesssible menus and more options for the new game/character customization menu and also fixes to the Read Tile feature.

In case you didn't knew, here's the link to the server - https://discord.gg/yQjjsDqWQX

1) Maybe i will, after the mod has been released though.

2) Yeah, those menu require a feedback like clicking on a button or entering the name of pet etc. Can't really change that but will make those menus accessible.

3) I actually don't understand this one.

4) Yup, i will but again, after the mod has been out of beta unfortunately.

5) That's interesting, i haven't encountered that bug yet, maybe it's because the game is in windowed mode cause many times small bugs like these happen because of windowed mode. Try switching to fullscreen.

6) I will, at some point in the future.

7) Hmm...I think I have an idea how to fix that. Will inform you if I have implemented it.

And if you're okay with it, maybe we can close this issue as it is a conversation.
